### PR TITLE
fix code for cloning repo

### PR DESCRIPTION
**Pull request by DelazJ**
I couldn't clone the repo using the code. Got:

> The authenticity of host 'github.com (192.30.252.128)' can't be established.
> RSA key fingerprint is 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48.
> Are you sure you want to continue connecting (yes/no)? yes
> Warning: Permanently added 'github.com,192.30.252.128' (RSA) to the list of known hosts.
> Permission denied (publickey).
> fatal: Could not read from remote repository.

Using the code in the PR just copies the repo without problem.

### CODE REVIEW

**Comment by DelazJ** [comment]
Wonder if this fix is even the right way given that

> git remote -v
> origin  git://github.com/qgis/QGIS-Documentation.git (fetch)
> origin  git://github.com/qgis/QGIS-Documentation.git (push)

while imho it should return `qgis/QGIS-Documentation` as upstream. Should the fix be 

> git clone git://github.com/Your-Username/QGIS-Documentation.git

as stated in[ Github help](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)? And I get what I expected. Can anyone more familiar with the process can give the right steps (fork, clone?) for any contributor (and not only those with write access), please?
Thanks

**Comment by yjacolin** [comment]
@DelazJ why did you expect to get upstream instead of origin? The default name for your local repository will always be origin when you clone a repository.

To get upstream (or whatever name you gave) you should add it: `git remote add upstream blabla`. This is useful only if you fork the repository into your account and whant to keep a link to the upstream repository.

**Comment by yjacolin** [comment]
LGTM anyway.

**Comment by DelazJ** [comment]
@yjacolin what looks good to you, git cloning qgis/QGIS or git cloning username/QGIS repo?

> @DelazJ why did you expect to get upstream instead of origin? The default name for your local repository will always be origin when you clone a repository.
> 
> To get upstream (or whatever name you gave) you should add it: git remote add upstream blabla.

Indeed, you are right. Sorry I'm mixing everything :(

> This is useful only if you fork the repository into your account and whant to keep a link to the upstream repository.

Isn't this how a contributor should work? Aren't that the expected steps to contribute for any user (also stated in your [how to contribute recommendations](https://github.com/yjacolin/QGIS-Documentation/wiki/howto_contribute_QGIS-Doc)):
1. make a fork of the repo in my github account 
2. (optional) make a local copy of myaccount/myrepo on my drive to be able to use command line tools
3. make changes in my account repo (the local or the online, keeping them synced)
4. and send PR from my online github account to the qgis repo?

If I clone, let's say the qgis/QGIS repo using `git clone git://github.com/qgis/QGIS.git` and assuming that I don't have write access to qgis/QGIS, where will I push my local changes and how can I create PR ?

File: source/site/getinvolved/development/qgisdevelopersguide/git.rst [hunk 1]
```diff
@@ -101,7 +101,7 @@ If you're interested in checking out QGIS documentation sources:
 
 .. code-block:: bash
 
-  git clone git@github.com:qgis/QGIS-Documentation.git
+  git clone git://github.com/qgis/QGIS-Documentation.git
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/git.rst:104]: why did you change this? This is not wrong! Both are correct anyway ;)

**Comment by DelazJ** [comment]
Yes, but I think it requires a special mode of connexion (which is not the default one -at least with github), right?
Anyway, I think this PR should be closed because:
- I do not well identify who is the target of this chapter (beginners, advanced)
- and I do not have a good picture of git processes. I "learn" git through GitHub steps so it might be a narrowed view and I 'm not sure this is the way things go.

**Comment by yjacolin** [comment]
Ok let's see how to improve this part for the next hackfest. Hope you can be there ;)
